src/Sylius/Bundle/ShopBundle/EventListener/UserCartRecalculationListener.php

```diff
@@ -11,7 +11,7 @@
 
 declare(strict_types=1);
 
-namespace Sylius\Bundle\CoreBundle\EventListener;
+namespace Sylius\Bundle\ShopBundle\EventListener;
```

Comment:
I wonder because nothing has changed with this service. Shouldn't we limit this service only to ShopSection requests? @lchrusciel, what was your idea for a refactor?

Reply:
We should, thanks for catching it up